store-conf/conf/msgs/ZsMsg.properties

```diff
@@ -938,3 +938,19 @@ verifyRecoveryEmailBodyHtml =\
   Recovery email verification code: {0}</br>\
   The code expires by {1}\
   </div>
+
+# password reset code verification
+sendPasswordRecoveryEmailSubject = Request to recover account using recovery email address by {0}
```

Comment:
Following format is mentioned in ticket ZCS-4781. Better to use the same template for the mail.
Subject: Reset your %DOMAIN% password

We have received a password reset request for your account, %EMAIL%.

Temporary Reset Code: X X X X X X X X
Use by: %EXPIRY%

Reply:
done